src/ae/aepp.ts

```diff
@@ -22,23 +22,27 @@
  * @example import { RpcAepp } from '@aeternity/aepp-sdk'
  */
 
-import Ae from './'
-import Aens from './aens'
-import Contract from './contract'
+import Ae from '.'
+import Aens, { _Aens } from './aens'
+import Contract, { _Contract } from './contract'
+// @ts-expect-error
 import AeppRpc from '../utils/aepp-wallet-communication/rpc/aepp-rpc'
-import Tx from '../tx/tx'
-import Oracle from './oracle'
+import TxStamp from '../tx/tx'
+import Tx from '../tx'
+import Oracle, { _Oracle } from './oracle'
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import stampit from '@stamp/it'
+
+export type Aepp = typeof Tx & _Aens & _Contract & _Oracle
```

Comment:
Ae type isn't mentioned here. But is being used in the stamp. This causes errors like `Property 'spend' does not exist on type 'Aepp'. Did you mean 'send'?`.